### PR TITLE
Implemented a Wallet

### PR DESCRIPTION
**Pull request by cyberdr0id**

### CODE REVIEW

File: btc_test.go [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+			wallet:          &Wallet{balance: 100.345},
+			withdrawSum:     100.345,
+			expectedBalance: 0,
+		},
+		{
+			wallet:          &Wallet{balance: 100.345},
+			withdrawSum:     50,
+			expectedBalance: 50.345,
+		},
+	}
```
Comment (papahmsolo) [inline btc_test.go:35]: - let's also add tests for negative cases, such as for errors
- you might find _subtest_ useful to run the test cases and we can give them names to describe the purpose of the test

File: btc.go [hunk 2]
```diff
@@ -0,0 +1,44 @@
+package btc
```
Comment (papahmsolo) [inline btc.go:1]: Package comment?

**Review by papahmsolo — CHANGES_REQUESTED** [review]
Cool!

Can you address these points:
- add `.gitignore`
- try running `golangci-lint` and you can also commit it's config to the repo

File: btc.go [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+
+import (
+	"errors"
+)
+
+// ErrNegativeSum presents custom error.
+var ErrNegativeSum = errors.New("Sum cannot be negative")
+
+// ErrInsufficientFunds presents custom error.
+var ErrInsufficientFunds = errors.New("Insufficient funds")
```
Comment (papahmsolo) [inline btc.go:12]: Let's put more meaningful comments

**Review by papahmsolo — CHANGES_REQUESTED** [review]
Tests look good 👍
Let's fix lint errors, files are not formatted, etc

**Review by papahmsolo — APPROVED** [review]
